src/components/ClipboardAwarePasteButton.tsx

```diff
@@ -30,14 +39,54 @@ export default function ClipboardAwarePasteButton({
     return null
   }
 
+  // Debounce onPress event so that it is called once on trigger and
+  // consecutive calls in given period are ignored.
+  const debouncedOnPress = debounce(
```

Comment:
I included it since I just copied over what was used in the generic button prop, removed it in [0ec330a](https://github.com/valora-inc/wallet/pull/5628/commits/0ec330a29c9ba037741e99d50e72a18f0c27f81a)